dd-sdk-android/src/main/kotlin/com/datadog/android/DatadogConfig.kt

```diff
@@ -384,6 +389,15 @@ private constructor(
             val providers = customProviders + defaultProviders
             return DatadogGesturesTracker(providers)
         }
+
+        companion object {
+            internal const val RUM_NOT_INITIALISED_WARNING_MESSAGE =
```

Comment:
maybe we should use a local string and copy paste it in the test method also ? Will avoid creating this companion object and also exposing it in the `apiSurface` ?